docs/source/api/letsencrypt_autorenew.rst

```diff
@@ -0,0 +1,76 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-letsencrypt-autorenew:
+
+*************************
+``letsencrypt/autorenew``
+*************************
+
+.. versionadded:: 1.4
+
+``POST``
+========
+Generates an SSL certificate and private key using Let's Encrypt for a :term:`Delivery Service`
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+No parameters available
+
+
+Response Structure
+------------------
+:LetsEncryptExpirations: A list of objects with information regarding certificate expiration for all Let's Encrypt certificates
+
+	:XmlId:       The :term:`Delivery Service`'s uniquely identifying 'xml_id'
+	:Version:     The version of the certificate record in Traffic Vault
+	:Expiration:  The expiration date of the certificate for the :term:`Delivery Service`
+	:AuthType:    The authority type of the certificate for the :term:`Delivery Service`
+	:Error:       Any errors received in the renewal process
+
+:SelfSignedExpirations:  A list of objects with information regarding certificate expiration for all self signed certificates
+:OtherExpirations:       A list of objects with information regarding certificate expiration for all other certificates
```

Comment:
These two still need format explanations. Alternatively, if this is just separating the expirations by AuthType and they all have the same structure, maybe you want to make them all one top-level array since the AuthType field already makes that distinction?

Reply:
it is just separating them by AuthType but they are all treated differently / presented separately so i would like to keep them separated.  I can add the format explanations for each if that is the right way to do it

Reply:
it is, if that's what you think makes the most sense.